docs/reference.rst

```diff
@@ -1,13 +1,63 @@
-The pygfx reference
-===================
+API Reference
+=============
 
+.. rubric:: Public API
 
-Resources
----------
+The primary way of accessing pygfx is by using the members of its top-level namespace.
+Currently this includes the following classes, which comprise the public API:
+
+.. autosummary::
+
+    pygfx.objects.WorldObject
+    pygfx.utils.color.Color
```

Comment:
Can we name these just `pygfx.WorldObject` etc?

Reply:
> It should be possible to create alias names or get autosummary to show a different name in the result

That would be great. That `pygfx.utils.color.Color` is rather ugly in hindsight 😄

Reply:
I looked into this over the weekend and there are two options:

1. We can hide the module in the autosummary table. This would turn `pygfx.utils.color.Color` into `Color`. Autosummary supports this out-of-the-box, and this is how I solved it for now.
2. We can add a small custom sphinx extension that extends autosummary by adding a custom prefix to the name in the table, i.e., it would turn `pygfx.utils.color.Color` into `pygfx.Color`.

In both cases the result would link to the same page that we use in the sub-package docs, so either solution works. Do you have any preference @almarklein ?

Reply:
Let's use the builtin option and use `Color`.